.ci/.e2e-tests-beats.yaml

```diff
@@ -0,0 +1,55 @@
+---
+SUITES:
+  - suite: "helm"
+    provider: "docker"
+    scenarios:
+      - name: "APM Server"
+        tags: "apm-server"
+        platforms: ["debian_amd64"]
+      - name: "Filebeat"
+        tags: "filebeat"
+        platforms: ["debian_amd64"]
+      - name: "Metricbeat"
+        tags: "metricbeat"
+        platforms: ["debian_amd64"]
+  - suite: "fleet"
```

Comment:
Is the base for this from https://github.com/elastic/e2e-testing/blob/main/.ci/.e2e-tests.yaml?

Can we maybe reference that file in a comment here, and note what the differences are and why?

Reply:
About the differences, I'd like the product team to be the ones to define what to run when beats triggers the e2e for Fleet. I could see it defining different profiles: for kubernetes autodiscover, metricbeat, filebeat, etc.